packages/plugins/max-depth/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @escape.tech/graphql-armor-max-depth
 
+## 2.3.0
+
+### Minor Changes
+
+- 32db0f2: fix: Max-Depth now throws a graphql error rather than syntax error issue [#572](https://github.com/Escape-Technologies/graphql-armor/pull/572)
+
```

Comment:
This is auto generated by running `yarn changeset`

Reply:
I have updated this & removed the manual update of the package.json version too, Very cool. Might have to add something similar to our projects.